backend/apis/parse-docket-pdf.utils.js

```diff
@@ -84,8 +84,16 @@ function parseCharges(lines, sections) {
   }
 
   walkSection(lines, sections, "CHARGES", addDispositionToCharges);
+  let chargesWithSeverity = charges.map(elm => {
+    let index = elm.description.search(/.[A-Z][a-z]|.\d[a-z]/);
+    let offenseName = elm.description.slice(0, index);
+    let severity = elm.description.slice(index + 1);
+    elm = { ...elm, offenseName, severity };
```

Comment:
Great work. I would suggest just creating a new object and then spreading elm to it instead of mutating the `elm` object with a new one.

ex:
```javascript
const { description, ..rest } = elm;
const index = description.search(/.[A-Z][a-z]|.\d[a-z]/);
const offenseName = description.slice(0, index);
const severity = description.slice(index + 1);
const newElm = { ...rest, offenseName, severity };
return newElm

```

also, not entirely sure what elm is, is it a charge?

Reply:
👍 agreed

Reply:
Sweet I like that better! thanks @scotttball I will make changes